include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -161,9 +161,9 @@ class SplineBuilder
     // interpolator specific
     std::unique_ptr<ddc::detail::Matrix> matrix;
 
-public:
     int compute_offset(interpolation_domain_type const& interpolation_domain);
 
+public:
```

Comment:
I agree with this change but I still think it is worth documenting this function even if only briefly so we remember what it is for. There is an explanation inside the function definition:
```suggestion
    /// Calculate offset so that the matrix is diagonally dominant
    int compute_offset(interpolation_domain_type const& interpolation_domain);

public:
```

Reply:
Can you clarify an offset of what ?